src/components/todoFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Filters } from '../types/Filters';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  currentFilter: Filters;
+  onFilterChange: (filter: Filters) => void;
+  todos: Todo[];
+  activeTodosCount: number;
+  onClearCompleted: () => void;
+}
+
+export const TodoFilter: React.FC<Props> = ({
+  currentFilter,
+  onFilterChange,
+  todos,
+  activeTodosCount,
+  onClearCompleted,
+}) => {
+  const completedTodosCount = todos.length - activeTodosCount;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodosCount} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${currentFilter === Filters.All ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={() => onFilterChange(Filters.All)}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={`filter__link ${currentFilter === Filters.Active ? 'selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={() => onFilterChange(Filters.Active)}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={`filter__link ${currentFilter === Filters.Completed ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
+          onClick={() => onFilterChange(Filters.Completed)}
+        >
+          Completed
+        </a>
```

Comment:
Consider using `Object.values(your enum)` and `map()` here

Reply:
Not fixed 
<img width="961" alt="image" src="https://github.com/user-attachments/assets/ca8f7249-8a5d-4beb-91a9-42c105a9c87d">